docs/configuring-well-known.md

```diff
@@ -50,15 +50,15 @@ matrix_static_files_file_matrix_support_enabled: true
 # Homeserver admin contacts as per MSC 1929 https://github.com/matrix-org/matrix-spec-proposals/pull/1929
 matrix_static_files_file_matrix_support_property_m_contacts:
   - matrix_id: "@admin1:{{ matrix_domain }}"
-    email_address: admin@domain.tld
+    email_address: admin@example.com
     role: m.role.admin
   - matrix_id: "@admin2:{{ matrix_domain }}"
-    email_address: admin2@domain.tld
+    email_address: admin2@example.com
     role: m.role.admin
-  - email_address: security@domain.tld
+  - email_address: security@example.com
     role: m.role.security
 
-matrix_static_files_file_matrix_support_property_m_support_page: "https://example.domain.tld/support"
+matrix_static_files_file_matrix_support_property_m_support_page: "https://matrix.example.com/support"
```

Comment:
This change should also be fine, though I am not perfectly sure.

Reply:
This property points to the "support page".

The example URL points to the base domain (or some other domain) which hosts your organization's support page (if any).

More details can be found on this MSC page: https://github.com/matrix-org/matrix-spec-proposals/blob/hs/proposal-admin-contact-1/proposals/1929-admin-contact.md

So I'd say `example.com/support` is a better address, to clearly indicate that it's not the Matrix homeserver that is hosting this page. You need to host it yourself somewhere.